.gcloudignore

```diff
@@ -18,5 +18,8 @@ __pycache__/
 # Ignored by the build system
 /setup.cfg
 
-# Secrets needed to deploy:
-!monorail-key.json
+# Secrets for local development only
+monorail-key.json
```

Comment:
You can remove this, it's in `.gitignore` and a negation was needed only to counteract that. For the same reason the `webapp/node_modules` entry ought also not be needed.

Reply:
But we do want to skip the things in .gitignore, all except one thing at least, previously two.

Reply:
This is the wording that made me think that .gitignore is automatically includes .gitignore:
```
# If you would like to upload your .git directory, .gitignore file or files
# from your .gitignore file, remove the corresponding line
# below:
```

I suspect that is wrong, did you experiment to confirm?

Reply:
Yes, and I just checked again.

The comment probably isn't wrong: the example template in the [doc](https://cloud.google.com/sdk/gcloud/reference/topic/gcloudignore) has the `#!include:.gitignore` line. I've added it back to our `.gcloudignore` so that it does honour `.gitignore`.